### PR TITLE
i18n(es): Translated `v2.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Following the translation roadmap I translated migration guide`v2.mdx` to spanish.

### CODE REVIEW

File: src/content/docs/es/guides/upgrade-to/v2.mdx [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+Esta guía te ayudará a migrar de la versión Astro v1 a la Astro v2.
+
+¿Necesitas actualizar un proyecto más antiguo a la v1? Ve nuestra [guía de migración anterior](/es/guides/upgrade-to/v1/).
+
+## Actualiza Astro
+
+Actualiza la versión de tu proyecto Astro a la última versión usando el manejador de paquetes. Si estás utilizando alguna integración, también acutalizalas a la última versión.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/upgrade-to/v2.mdx:15]:
```suggestion
Actualiza la versión de tu proyecto Astro a la última versión usando el gestor de paquetes. Si estás utilizando alguna integración, también acutalizalas a la última versión.
```

File: src/content/docs/es/guides/upgrade-to/v2.mdx [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+
+### Eliminado: Soporte para Node 14
+
+Node 14 está programado para llegar al final de su vida úitl en Abril de 2023.
+
+Astro v2.0 elimina completamente el soporte para Node 14, para que todos los usuarios de Astro puedan aprovechar las caraacterísticas más modernas de Node.
+
+#### ¿Qué debería hacer?
+
+Checa que tanto tu entorno de desarrollo como tu entorno de producción estén usando **Node `16.12.0` o superior**.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/upgrade-to/v2.mdx:61]:
```suggestion
Verifica que tanto tu entorno de desarrollo como tu entorno de producción estén usando **Node `16.12.0` o superior**.
```

File: src/content/docs/es/guides/upgrade-to/v2.mdx [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+
+1. Checa la versión local de Node usando:
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/upgrade-to/v2.mdx:63]:
```suggestion
1. Verifica la versión local de Node usando:
```

File: src/content/docs/es/guides/upgrade-to/v2.mdx [hunk 4]
```diff
@@ -0,0 +1,421 @@
...
+
+    ```sh
+    node -v
+ 
+    ```
+    Si tu entorno de desarrollo necesita actualizarse, [instala Node](https://nodejs.org/es/download/).
+
+2. Checa la propia documentación del [entorno de producción](/es/guides/deploy/) para verificar que ellos soportan Node 16.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/upgrade-to/v2.mdx:71]:
```suggestion
2. Verifica la propia documentación del [entorno de producción](/es/guides/deploy/) para verificar que ellos soportan Node 16.
```

File: src/content/docs/es/guides/upgrade-to/v2.mdx [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+
+#### Agregado: `gfm` y `smartypants`
+
+En la versión v1.x, podías elegir deshabilitar ambos plugins predeterminados de Markdown de Astro (GitHub-Flavored Markdown y SmartyPants) configurando `markdown.extendDefaultPlugins: false`.
+
+Astro v2.0 remplaza `markdown.extendDefaultPlugins: false` con opciones booleanas separadas para controlar individualmente cada uno de los plugins predeterminados de Markdown de Astro. Estos están habilitados por defecto y se pueden configurar como `false` de forma independiente.
+
+##### ¿Qué debería hacer?
+
+Elimina `extendDefaultPlugins: false` y agrega las flags para deshabilitar cada plugin individualmente en su lugar.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/upgrade-to/v2.mdx:157]:
```suggestion
Elimina `extendDefaultPlugins: false` y agrega las opciones para deshabilitar cada plugin individualmente en su lugar.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
gracias @dreyfus92 deje unos comentarios

**Comment by dreyfus92** [comment]
Gracias por el review @kevinzunigacuellar  😄

**Pull request merged**